tests/unit/fake_data_root/sos_commands/ceph/ceph_osd_crush_dump

```diff
@@ -90,7 +90,12 @@
                     "id": -7,
                     "weight": 6396,
                     "pos": 2
-                }
+                },
+                {   
+                    "id": -9, 
+                    "weight": 6396,
+                    "pos": 3
+                }   
             ]
```

Comment:
I would like if possible to keep the contents of this file "meaningful" since they are currently based on a real ceph cluster that is the same as the other files in the sos_commands/ceph dir. To do this kind of this i would either add comments either side of the additional text (it's json so should ok to add # comments) or put it inline in the unit test (which might not work so well here since its a lot of text).

Reply:
hey @pponnuvel i see you moved this to the unit test code so I assume this change can be removed?

Reply:
@dosaboy Was still making changes (one more test). Done now & ready.

Reply:
ah sorry about that, looks better now. One last thing on the unit test, ill comment inline.